torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -10,6 +10,7 @@
 get_dimensions_image_pil = _FP.get_dimensions
 
 
+# TODO: Should this be prefixed with `_` similar to other methods that don't get exposed by init?
```

Comment:
Right now, all helper methods in all subpackages usually start with `_`. This is not about defining the scope, since all the subpackages are private. This is mostly about being consistent on the naming patterns. For example while validating that all kernels are being called by my utility on tests, I started debugging why `get_chw` wasn't selected only to realize it's because it's never imported and made public in `__init__.py`. Since this method is used in multiple files, this should happen in a separate PR if we decide to do it.